### ISSUE TITLE
Error - No code in project

### ISSUE THREAD

**Issue by 4val0v** [exakat/exakat#43]
root@server:~/exakat# php exakat.phar init -p ES51
root@server:~/exakat# php exakat.phar project -p ES51

Error : No code in project 'ES51' 
on file phar:///root/exakat/exakat.phar/[library/Exakat/Tasks/Files.php
on line 57](https://github.com/exakat/exakat/blob/master/library/Exakat/Tasks/Files.php#L57)

Because of what this error?

_cd projects/ES51/ && ls_
![ls](https://i.imgur.com/iZwb5r3.png)

**Comment by exakat** [comment]
Exakat has a configuration that ignore some files : check /projects/ES51/config.ini.

Also, Exakat ignores automatically files that can't compiles. In some situations, if the code was for PHP 5.6, and it is being audited with 7.1, no file compile. 

I'll see if I can add a better message for that.

**Comment by 4val0v** [comment]
The project is made on PHP5
On the server I checked with PHP 7.2

Installed other versions 5.5 and 5.6

_exakat.ini_
```
php55 = /usr/bin/php5
php56 = /usr/bin/php5.6
;php70 = /usr/local/sbin/php70
;php71 = /usr/local/sbin/php71
;php72 = /usr/local/sbin/php72
;php73 = /usr/local/sbin/php73
php72 = /usr/bin/php
```

Console out:
`php exakat.phar project -p evo51 //PHP 7.2`
```
Error : No code in project 'evo51' 
on file phar:///root/exakat/exakat.phar/library/Exakat/Tasks/Files.php
on line 57
```
`php5 exakat.phar project -p evo51`
```
PHP Parse error:  syntax error, unexpected '{', expecting identifier (T_STRING) in phar:///root/exakat/exakat.phar/library/Exakat/Config.php on line 25
```
`php5.6 exakat.phar project -p evo51`
```
PHP Parse error:  syntax error, unexpected '{', expecting identifier (T_STRING) in phar:///root/exakat/exakat.phar/library/Exakat/Config.php on line 25
```
---
**In the project directory should only be php scripts?**
_css, sql, txt, js, .htaccess, .md_ **files need to be ignored or deleted?**

**Comment by 4val0v** [comment]
https://github.com/exakat/exakat/blob/master/library/Exakat/Tasks/Files.php#L56
The source code of the project should be on this path?
~/exakat/projects/evo51/**code** ?
or
~/exakat/projects/evo51/
?????

thx

**Comment by Mte90** [comment]
I have the same issues of projects not found.
Is not clear to me because in case of errors is not written what are the path where the tool is looking for.
Maybe write the paths will be helpful on debug.

**Comment by Mte90** [comment]
In my case was a symlinked folder that was not available from inside a vm.

**Comment by exakat** [comment]
Is this still on ? I would like to close it, if it doesn't happen anymore.

**Comment by Mte90** [comment]
I am not using anymore exakat so I don't know.

**Comment by justicecloney** [comment]
I'm having this exact issue, and this is the only place on the internet that even mentions the error.
i started with 
`php exakat.phar init -p <myrepo> -R 'https://gitub.com/<myuser>/<myrepo>.git'`
then 
`php exakat.phar project -p <myrepo>`
and it gives no code in project. Going into the directory, it didn't download the git (obvious by the instant init).
browsing to the folder and finding only the sql file and basic init, i copied the code in and ran it again, nothing. I checked the config.ini, nothing weird. I manually included the main folder of code into the config, no change. I tried having the .git folder at that root of the project folder, that didn't work.
